components/teleport/Teleport.vue

```diff
@@ -64,7 +64,7 @@
           v-model="amount"
           root-class="w-full"
           input-class="pr-2"
-          step="0.01"
+          step="0.00001"
```

Comment:
> but teleporting dust does not make sense either.

please set some sensible minimum value
maybe 0.1?

Reply:
keep it 0.01 we had issue in the past where user wanted to teleport less than 0.1 on Kusama

Reply:
I increased it bcz of this 

![CleanShot 2023-11-28 at 14 05 06](https://github.com/kodadot/nft-gallery/assets/44554284/8c46c8a1-9a0e-4c09-a7d2-3aa7a93870db)

should max balance be 0.46 in this case instate of all of it 

![CleanShot 2023-11-28 at 14 05 42@2x](https://github.com/kodadot/nft-gallery/assets/44554284/3d12af25-4b78-4e71-ab44-3ec3f956798b)

Reply:
>I increased it bcz of this

yes that's the step size

but we need some  min > 0 to avoid sending dust
so 0.01 it is i guess